autobuild/mac/artifacts/autobuild_mac_2_build.sh

```diff
@@ -7,6 +7,12 @@
 ###  PARAMETERS  ###
 ####################
 
+SIGN=$1
+echo "${SIGN}"
+if [ -n "${SIGN}" ]; then
+    shift
```

Comment:
I think I am not really happy here (if I understand this correctly). You’re shifting all parameters to the left if the first argument is empty? What would happen if we add new parameters to the script (e.g. for build options as we did for Windows?)
I‘d really prefer the -s option…

Reply:
See previous comment as to why this is necessary

Reply:
I probably need to live with it.

Reply:
I think you misunderstood what this does.
```
-n
   string is not null.
```
We shift if there _is_ an argument, so as to not upset the rest of the script.

Reply:
Ok. Thanks. Didn't look up the bash documentation. Shame on me ;-(